nss-day-cohort-29/welcome-to-nashville-valkyries
# Ab restaurants

# Description

Added restaurants-api.js. Fetches from Zomato API based on search variable and returns strings of the top 20 results with:
1. Name of Restaurants
2. Address
3. Rating


## Type of change

Please delete options that are not relevant.



# Testing Instructions for Change Made

--git fetch all

--git checkout ab-restaurants

open code

check to see if console log's list of restaurants with Name, Address and Rating.

to test different cuisines, change "searchVal" value to mimic search


# Checklist:
